csharp_dotnetcore/60.ConversationHistory/TranscriptStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Bot.Builder.Azure;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public class TranscriptStore
+    {
+        public TranscriptStore(AzureBlobTranscriptStore store)
+        {
+            Store = store;
+        }
```

Comment:
Null check + throw.

Reply:
Not applicable anymore.

---
In reply to: [218236713](https://github.com/Microsoft/BotBuilder-Samples/pull/391#discussion_r218236713) [](ancestors = 218236713)